examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/OfferingFragment.kt

```diff
@@ -59,9 +59,13 @@ class OfferingFragment : Fragment(), PackageCardAdapter.PackageCardAdapterListen
 
     override fun onPurchasePackageClicked(cardView: View, currentPackage: Package) {
         binding.purchaseProgress.visibility = View.VISIBLE
-        Purchases.sharedInstance.purchasePackageWith(
+        val basePlanOption = currentPackage.product.purchaseOptions.first { it.isBasePlan }
```

Comment:
maybe this should be an extension function..

Reply:
Well if we do, I think it would be short term. I think we won't need to access the base plan option that much in the future. Ideally, the UI in purchase tester allows to see all the purchase options and allows to choose one but this can be done later (lmk if you would like me to tackle this part)

Reply:
I think this is something we expect devs to do a lot? If it is, maybe we could even have a property `basePlan` added to the StoreProduct. Although I assume folks would probably prefer to offer a better offer rather than the base plan

Reply:
>  Ideally, the UI in purchase tester allows to see all the purchase options and allows to choose one but this can be done later (lmk if you would like me to tackle this part)

@tonidero sure, if you want to take that on, that'd be great. maybe once we see the actual code behind displaying options, we'll have a better idea of what the API should be. i was mostly just making updates so i could test postReceipt at this point.

Reply:
Sounds good, i can work on that tomorrow. I can take over the Jira task once you're done.

Reply:
just fyi i also created a ticket for updating magicweather, which can have a more polished paywall... idk if we want to do that sooner than later too